src/main/java/core/basesyntax/WriteData.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteData {
+    public void writer(String toFile,String dataToWrite) {
```

Comment:
bad class name - you should not use verbs

how about `FileDataWriter`?

and considering that you could simplify the method name to `public void writeTo(String toFile, String data)`